static/app/views/alerts/rules/metric/triggers/anomalyAlertsForm.tsx

```diff
@@ -0,0 +1,156 @@
+import {Component, Fragment} from 'react';
+import styled from '@emotion/styled';
+
+import SelectControl from 'sentry/components/forms/controls/selectControl';
+import FieldGroup from 'sentry/components/forms/fieldGroup';
+import {
+  AlertRuleSensitivity,
+  AlertRuleThresholdType,
+  type UnsavedMetricRule,
+} from 'sentry/views/alerts/rules/metric/types';
+
+type Props = {
+  disabled: boolean;
+  onSensitivityChange: (sensitivity: AlertRuleSensitivity) => void;
+  onThresholdTypeChange: (thresholdType: AlertRuleThresholdType) => void;
+  sensitivity: UnsavedMetricRule['sensitivity'];
+  thresholdType: UnsavedMetricRule['thresholdType'];
+  /**
+   * Map of fieldName -> errorMessage
+   */
+  error?: {[fieldName: string]: string};
+
+  hideControl?: boolean;
+};
+
+type SensitivityFormItemProps = {
+  onSensitivityChange: (sensitivity: AlertRuleSensitivity) => void;
+  sensitivity: UnsavedMetricRule['sensitivity'];
+};
+
+type DirectionFormItemProps = {
+  onThresholdTypeChange: (thresholdType: AlertRuleThresholdType) => void;
+  thresholdType: UnsavedMetricRule['thresholdType'];
+};
+
+function SensitivityFormItem({
+  sensitivity,
+  onSensitivityChange,
+}: SensitivityFormItemProps) {
+  const handleSensitivityChange = ({value}) => {
+    onSensitivityChange(value);
+  };
+
+  return (
+    <StyledField
+      label={<div>{'Sensitivity'}</div>}
+      help={
+        <div>
+          {
```

Comment:
nit: could you just pass in the string or do you need the display block css setting from the div?